htmldocx/h2d.py

```diff
@@ -159,7 +192,7 @@ def delete_paragraph(paragraph):
 
 class HtmlToDocx(HTMLParser):
 
-    def __init__(self):
+    def __init__(self, ul_style=None, ol_style=None):
```

Comment:
why not set defaults here rather than through or?

Reply:
I moved them up here like you suggested.  I also added table_style and paragraph_style in the args too, since they are in the init method.